### PR TITLE
Fix typos & update directions for setting the Firebase project ID

### PR DESCRIPTION
**Pull request by jeffpaul**
## Summary

This PR is a placeholder for docs updates after testing staging.wptide.org's local install steps.

Fixes #

## Checklist

- [ ] My pull request is addressing an [open issue](https://github.com/wptide/wptide.org/issues) (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/wptide/wptide.org/tree/develop/web/docs/contributing#tests).
- [x] My code follows the [Tide Contributing Guide](https://github.com/wptide/wptide.org/tree/develop/web/docs/contributing) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: web/docs/installation/README.md [hunk 1]
```diff
@@ -77,12 +77,14 @@ Copy the hidden files:
...
 ::: tip IMPORTANT
 
-This command should only be used once. The new `.env` is for the Firebase Functions API and the `.env.server` is for the Docker based Cloud Run Servers. Both files are for local development only and likely do not require any changes, except for the project ID that you created above.
-
-Also, make sure to update the `.firebaserc` with the project ID after this command is executed, as well.
+The following command should only be used once.
 
     npm run copy
 
+The new `.env` is for the Firebase Functions API and the `.env.server` is for the Docker based Cloud Run Servers. Both files are for local development only and likely do not require any changes, except for the project ID that you created above.
```
Comment (jeffpaul) [inline web/docs/installation/README.md]: It's unclear to me if I need to change anything in the `.env` or `.env.server` files here.  I'm going with no changes, but see that in `.env` and `.env.server` I should possibly be updating the value for `GOOGLE_CLOUD_PROJECT` to my Firebase project ID.  @derekherman which is correct?

**Comment by jeffpaul** [comment]
I ran through just the docs testing setup steps, no further tweaks to docs besides the small items in this PR plus the question on the .env and .env.server values possibly needed for the Firebase project ID.

File: web/docs/installation/README.md (hunk 1 above)
Comment (derekherman) [reply web/docs/installation/README.md]: @jeffpaul Yes, you would update to your project ID. Good catch.

**Comment by derekherman** [comment]
@jeffpaul I setting this back to draft, this is not done or ready for review. We have a lot of things not done in the docs.

**Comment by derekherman** [comment]
Hmm, actually. This is a PR you opened. Did you use an existing branch? I thought I had an open docs PR.

**Comment by derekherman** [comment]
@jeffpaul looks like I was thinking of #35 waiting on doing anything with this as GitHub actions are down and a deploy would fail.

File: web/docs/installation/README.md (hunk 1 above)
Comment (derekherman) [reply web/docs/installation/README.md]: `GOOGLE_CLOUD_PROJECT` is the same as the Firebase project ID they are one in the same.

Comment (derekherman) [reply web/docs/installation/README.md]: @jeffpaul did you want to make any changes to this knowing what you know now?

Comment (derekherman) [inline web/docs/installation/README.md]:
```suggestion
The new `.env` is for the Firebase Functions API and the `.env.server` is for the Docker based Cloud Run Servers. Both files are for local development only and likely do not require any changes, except for the project ID that you previously created should now be used as the value for the `GOOGLE_CLOUD_PROJECT` environment variable.
```

File: web/docs/installation/README.md [hunk 2]
```diff
@@ -77,12 +77,14 @@ Copy the hidden files:
...
+
+Also, make sure to update `.firebaserc` with the project ID after this command is executed, as well. The project ID will be the value associated to the `default` key in `.firebaserc`.
```
Comment (derekherman) [inline web/docs/installation/README.md]:
```suggestion
Also, make sure to update `.firebaserc` with the project ID after this command is executed. The project ID will be the value associated to the `default` key in `.firebaserc`.
```

**Pull request merged**